src/test/java/com/vikingzorros/rehabit/repository/UserRepositoryTest.java

```diff
@@ -0,0 +1,132 @@
+package com.vikingzorros.rehabit.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import com.vikingzorros.rehabit.daorepositories.PostRepository;
+import com.vikingzorros.rehabit.daorepositories.UserRepository;
+import com.vikingzorros.rehabit.entities.Post;
+import com.vikingzorros.rehabit.entities.User;
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@Slf4j
+@RunWith(SpringRunner.class)
+@DataJpaTest
+public class UserRepositoryTest {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private PostRepository postRepository;
```

Comment:
You should not use PostRepository seperately here,try to fetch post from user object itself

Reply:
Done

Reply:
Changes done.